docs/reference/mapping/types/nested.asciidoc

```diff
@@ -207,29 +214,20 @@ document as standard (flat) fields. Defaults to `false`.
 [float]
 === Limits on `nested` mappings and objects
 
-As described earlier, each nested object is indexed as a separate document under the hood.
-Continuing with the example above, if we indexed a single document containing 100 `user` objects,
-then 101 Lucene documents would be created -- one for the parent document, and one for each
+As described earlier, each nested object is indexed as a separate document.
```

Comment:
I think we either need to keep 'under the hood', or say 'a separate Lucene document'. Otherwise it this could be interpreted as each nested object corresponding to a new Elasticsearch document.

Reply:
I think that "indexed as a separate Lucene document" is more descriptive than "indexed as a separate document under the hood".